src/App.re

```diff
@@ -1,36 +1,31 @@
-type route =
-  | Login
-  | JobApp;
-
-type state = {route};
+type state = {token: string};
```

Comment:
Option(string) should be used here, but at this point I don't care

Reply:
addressed in latest commit